File: Easy/LCE - 1317.js
/**
 * @param {number} n
 * @return {number[]}
 */
var getNoZeroIntegers = function(n) {
    
    let i=1
    while(true) {
        if(check(n-i) && check(i))
            return [i, n-i]
        i++
    }
    
    function check(n) {
        return Array.from(n.toString()).indexOf('0') < 0
    }
    
};

File: Easy/Python - Solutions/LCE_1317.py
class Solution:
    def getNoZeroIntegers(self, n: int) -> List[int]:
        n1 = 1
        n2 = n-1

        while n1 != n:
            if n1+n2 == n and "0" not in str(n1) and "0" not in str(n2):
                return [n1, n2]
            else:
                n1 += 1
                n2 -= 1

        return []
